### ISSUE TITLE
Minor Glyph Adjustments

### ISSUE THREAD

**Issue by Marcus98T** [ChiuMing-Neko/ChiuKongGothic#9]
This one is minor, but doing it can improve the quality of the Japanese-style glyphs. [I have also posted a similar issue to Source Han Sans](https://github.com/adobe-fonts/source-han-sans/issues/293), but I think revising those glyphs will be slow on Adobe's part.

For now I will give two examples:
- 间 should include the feet only on the left side of 日, similar to the traditional counterpart 間. I circled where to add the feet in green.
- 啟 should have the right feet removed, at the area circled in red.

<img width="290" alt="Screenshot 2022-02-06 at 01 24 58" src="https://user-images.githubusercontent.com/10541683/152652312-d3bef049-64fc-4e97-9bde-8818efda4be0.png">

**Comment by ChiuMing-Neko** [comment]
Thank you for your suggestion.

1. Currently the major adjustment and redesigning of simplified characters (the 簡化字 under the definition of 簡化字總表) have been postponed due to the glyph form style of this font is more suitable for use in the Traditional Chinese context. I will adjust these simplified characters including the adjustment that you have mentioned in the range of GB2312 after I finished the work on BIG5(Commonly used), Cantonese commonly used characters and GB12345.  This means that the major update on the simplified characters will continue after both versions of the font can be used in daily Traditional Chinese context without any component glyph form inconsistency but ignoring some minor differences. 

2. I would rather keep the feet for the balancement of the glyph without adjusting the 口 and 攵 to rebalance it if the feet were removed.

3. For some reason, I would like to not alter the Adobe-Japan 1-6 Glyph and KR glyph, even though some of them have minor problems as you mentioned.  You may notice that the 「市(U+5E02)」that I had redesigned to distinguish from 「巿(U+5DFF)」 replace the CN glyph instead of the current CID place for JP glyph, and the JP glyph can still be accessed by using IVS.
![Image](https://user-images.githubusercontent.com/84143394/152669018-6af1735b-1e5d-4c89-ac36-a0b0fb512f2d.png)
But if Adobe makes an update on these glyphs, I will merge these updates in the next version of ChiuKong Gothic font.

4. The adjustment on 啥 will be included in the next update, the current design of 啥 is just some minor adjustment from CN glyph. But since the VF master of 1.004 is not released from Adobe, I am afraid that several re-added glyphs can not 100% reflect the original design.

**Comment by Marcus98T** [comment]
> I would rather keep the feet for the balancement of the glyph without adjusting the 口 and 攵 to rebalance it if the feet were removed.

I don't want to cause some misunderstanding, but I believe the Japanese type designers generally seem follow the rule that feet should not get in the way of the other components for like, I don't know, aesthetics, readability and less noisy reasons. As for number 2, 啟, I still think it's better to have the balance and keep it consistent with other JP glyphs by removing the right feet of 口, in other words, you can keep the left feet, but not the right feet, as to me, it seems to disturb the balance when 攵 is touching the right feet. Here is my mockup to show what it should look like:

<img width="317" alt="Screenshot 2022-02-07 at 00 59 38" src="https://user-images.githubusercontent.com/10541683/152691949-e35433b1-45ed-43f8-85b7-a4f02d74f182.png">

> Currently the major adjustment and redesigning of simplified characters (the 簡化字 under the definition of 簡化字總表) have been postponed due to the glyph form style of this font is more suitable for use in the Traditional Chinese context. 

> But if Adobe makes an update on these glyphs, I will merge these updates in the future version of ChiuKong Gothic font.

As for this, I am doing my part to make the glyphs look better on Adobe's end first, so I'm posting such issues to their font page.

> But since the VF master of 1.004 is not released from Adobe, I am afraid that several re-added glyphs can not 100% reflect the original design.

I understand this, which is why I never said 100%. I get the feeling that's how it is for having Adobe move to a variable workflow only after removing some additional JP glyphs which could have been useful. And Adobe for technical and business reasons still cannot release the overlapping source yet, it's been several years. And I think that I have to accept on my end that nothing is 100% perfect, at least whether it's worth adjusting some glyphs to make it 75% close to the 1.004 reference, it's up to you.

Only by Adobe releasing all the overlapping source files (including JP glyphs that didn't make it to v1), then this font could be more perfect, but too bad it's not gonna happen.

**Comment by ChiuMing-Neko** [comment]
> I don't want to cause some misunderstanding, but I believe the Japanese type designers generally seem follow the rule that feet should not get in the way of the other components for like, I don't know, aesthetics, readability and less noisy reasons. As for number 2, 啟, I still think it's better to have the balance and keep it consistent with other JP glyphs by removing the right feet of 口, in other words, you can keep the left feet, but not the right feet, as to me, it seems to disturb the balance when 攵 is touching the right feet. Here is my mockup to show what it should look like:
> 
> <img alt="Screenshot 2022-02-07 at 00 59 38" width="317" src="https://user-images.githubusercontent.com/10541683/152691949-e35433b1-45ed-43f8-85b7-a4f02d74f182.png">

Thank you for your clarification, the adjustment on 啟 will be included in the upcoming update.

**Comment by Marcus98T** [comment]
I saw the v1.3 update and I like to say congratulations on your effort. Some glyphs are 95% close to v1.004 which I deem good enough. Also I think 囔 is better balanced than the original v1.004 version (below).

<img width="197" alt="Screenshot 2022-10-16 at 14 25 20" src="https://user-images.githubusercontent.com/10541683/196021469-45b7a0fb-e3e6-47cc-977b-f39805c2df77.png">

However, I totally forgot to ask that 啟 may need to be adjusted again, just for balance. The 启 part (on the extralight master only) will need to stretch downwards a bit, similar to the CN glyph.

<img width="200" alt="Screenshot 2022-10-16 at 14 14 41" src="https://user-images.githubusercontent.com/10541683/196021139-aa8e18a0-f2dd-49ef-9451-d5b7574d78d0.png">

I also wonder if I can try to push some of the changes to the components of 吾, 南 and the JIS78/KR version of 翠 to Adobe itself. We might raise an issue soon but don't know whether they will accept any forks.

**Comment by ChiuMing-Neko** [comment]
> 啟

Noted, may adjust them later.

> I also wonder if I can try to push some of the changes to the components of 吾, 南 and the JIS78/KR version of 翠 to Adobe itself. We might raise an issue soon but don't know whether they will accept any forks.

For the part of remapping these components to match the JP glyph pattern just pretty simple, remapping them into the corresponding TW/HK glyph.

I believe that the problem with the 吾 and 南 components is from the fallback glyph mechanism in SHS by JP>KR>CN>TW>HK order, resulting in many glyphs inside of Adobe Japan 1-7 instead of following the JP glyph pattern and has its own JP glyph or using the TW/HK glyph as it fallback if they have shared the similar glyph form to Adobe Japan 1-7 Chart, they use this fallback order picking the CN glyph which has minor unnecessary diff. on details that break the JP glyph pattern. Which I think that details are pretty unnecessary to be deemed as the regional difference for CN glyph, as still many more CN glyphs still share the 五 shape with the JP glyph as well.
![圖片](https://user-images.githubusercontent.com/84143394/196030730-0b8d19fa-6c0c-4a05-b25b-2e10c2ba4a37.png)


And 南 component considering the fact that the extremely strict glyph shape and stroke restriction *(including several subtle rules) has to make the CN glyph individually, so I believe that they just mistakenly fall back to the CN glyph.
